<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="HYDNVC369" failures="2" tests="5" name="Free CRM App Test Cases" time="17.161" errors="0" timestamp="2021-04-02T21:13:36 IST">
  <testcase classname="com.qa.tests.LoginPageTest" name="@AfterMethod teardown" time="0.003">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.tests.LoginPageTest.teardown(LoginPageTest.java:108)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod teardown -->
  <testcase classname="com.qa.tests.LoginPageTest" name="@BeforeMethod initial" time="-1.617378208867E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod initial -->
  <testcase classname="com.qa.tests.LoginPageTest" name="@AfterMethod teardown" time="-1.617378208871E9">
    <skipped/>
  </testcase> <!-- @AfterMethod teardown -->
  <testcase classname="com.qa.tests.LoginPageTest" name="@BeforeMethod initial" time="-1.617378208873E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod initial -->
  <testcase classname="com.qa.tests.LoginPageTest" name="@AfterMethod teardown" time="-1.617378208877E9">
    <skipped/>
  </testcase> <!-- @AfterMethod teardown -->
  <testcase classname="com.qa.tests.HomePageTest" name="@AfterMethod teardown" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.tests.HomePageTest.teardown(HomePageTest.java:108)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod teardown -->
  <testcase classname="com.qa.tests.HomePageTest" name="@BeforeMethod login" time="-1.617378216343E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod login -->
  <testcase classname="com.qa.tests.HomePageTest" name="@AfterMethod teardown" time="-1.61737821635E9">
    <skipped/>
  </testcase> <!-- @AfterMethod teardown -->
  <testcase classname="com.qa.tests.HomePageTest" name="@AfterTest endReport" time="-1.617378216352E9">
    <skipped/>
  </testcase> <!-- @AfterTest endReport -->
  <testcase classname="com.qa.tests.LoginPageTest" name="@AfterTest endReport" time="-1.617378216355E9">
    <skipped/>
  </testcase> <!-- @AfterTest endReport -->
  <testcase classname="com.qa.tests.LoginPageTest" name="test" time="0.021">
    <failure type="java.lang.AssertionError" message="expected [Facebook] but found [Facebook – log in or sign up]">
      <![CDATA[java.lang.AssertionError: expected [Facebook] but found [Facebook – log in or sign up]
at com.qa.tests.LoginPageTest.test(LoginPageTest.java:85)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase classname="com.qa.tests.LoginPageTest" name="test2" time="0.0">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="com.qa.tests.LoginPageTest" name="test3" time="0.0">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="com.qa.tests.HomePageTest" name="test1" time="0.022">
    <failure type="java.lang.AssertionError" message="expected [welcome] but found []">
      <![CDATA[java.lang.AssertionError: expected [welcome] but found []
at com.qa.tests.HomePageTest.test1(HomePageTest.java:94)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="com.qa.tests.HomePageTest" name="test2" time="0.001">
    <skipped/>
  </testcase> <!-- test2 -->
</testsuite> <!-- Free CRM App Test Cases -->
